### ISSUE TITLE
Is it possible to have the latest Dotnet Core SDK on the microsoft/dotnet-framework-build?

### ISSUE THREAD

**Issue by jsacapdev** [Microsoft/dotnet-framework-docker#88]
Currently the microsoft/dotnet-framework-build has the following Dotnet Core SDK:

```
PS C:\> dotnet --info
.NET Command Line Tools (2.1.2)

Product Information:
 Version:            2.1.2
 Commit SHA-1 hash:  5695315371

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.2\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.3
  Build    : a9190d4a75f4a982ae4b4fa8d1a24526566c69df

```

Is it possible to get the latest version installed? i.e.

```
.NET Command Line Tools (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.4\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54
```

I would like a single docker container that has both the .NET Framework and .NET Core SDK (2.1.4) installed. At the moment I need to take microsoft/dotnet-framework-build, and then layer on the .NET Core SDK. 

If this is something that you would consider, I am happy to put forward a PR.

**Comment by MichaelSimons** [comment]
Yes is something that should be upgraded.  The VS components included in the build images are pointing to 15.4 artifacts.  They should get updated to 15.5 which will bring in the latest .NET Core SDK and runtime.

**Comment by jsacapdev** [comment]
The build tools link provides the latest .NET Core SDK and runtime. I have tested this by rebuilding 4.7.1-windowsservercore-ltsc2016 locally and I get 2.0.5 runtime and 2.1.4 sdk. I am guessing there isn't anything more for me to do (let me know if there is) and I can just wait for the images to be rebuilt and to be made available.
